Packs/Cybereason/Integrations/Cybereason/Cybereason_test.py

```diff
@@ -137,9 +138,10 @@ def test_malop_processes_command(mocker):
 
     args = {"malopGuids": None, "machineName": "desktop", "dateTime": "2022/08/01 00:00:00"}
     mocker.patch.object(demisto, 'results')
-    with pytest.raises(Exception):   
+    with pytest.raises(Exception):
         command_output = malop_processes_command(client, args)
```

Comment:
I suggest checking the content of the exception to make sure the function failed for the right reasons.

Reply:
Done